CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing to leaderboard-backend
+We appreciate your help!
+
+## Before filing an issue
+If you are unsure whether you have found a bug, please consider asking in [our discord](https://discord.gg/TZvfau25Vb) first.
+
+Similarly, if you have a question about a potential feature, [the discord](https://discord.gg/TZvfau25Vb) can be a fantastic resource for first comments.
```

Comment:
This could be a good use of [footnote link referencing referencing][linkref]. For example:

```suggestion
If you are unsure whether you have found a bug, please consider asking in [our discord][discord] first.

Similarly, if you have a question about a potential feature, [the discord][discord] can be a fantastic resource for first comments.
[discord]:https://discord.gg/TZvfau25Vb
```

[linkref]:https://github.github.com/gfm/#example-535